### PR TITLE
DP-2544-Add example to read all tabs from Excel file

### PR DESCRIPTION
**Pull request by chlocarlos**

### CODE REVIEW

File: src/main/java/com/northconcepts/datapipeline/examples/cookbook/ReadAllTabsFromAnExcelFile.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            DataReader reader = new ExcelReader(document).setFieldNamesInFirstRow(true).setSheetName(sheetName);
+            reader.open();
+            try {
+                Record record;
+                while ((record = reader.read()) != null) {
+                    log.info(record);
+                }
+            } finally {
+                reader.close();
+            }
```
Comment (gvjoshi25) [inline src/main/java/com/northconcepts/datapipeline/examples/cookbook/ReadAllTabsFromAnExcelFile.java]: @chlocarlos Please consider using this: `Job.run(reader, new StreamWriter(System.out));`
No need to manually open & close reader.
Also remove line no 20 for log declaration.

**Review by gvjoshi25 — CHANGES_REQUESTED** [review]

File: src/main/java/com/northconcepts/datapipeline/examples/cookbook/ReadAllTabsFromAnExcelFile.java (hunk 1 above)
Comment (chlocarlos) [reply src/main/java/com/northconcepts/datapipeline/examples/cookbook/ReadAllTabsFromAnExcelFile.java]: Done

File: src/main/java/com/northconcepts/datapipeline/examples/cookbook/ReadAllTabsFromAnExcelFile.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+import java.io.File;
+
+public class ReadAllTabsFromAnExcelFile {
+
+    public static void main(String[] args) throws Throwable {
+        ExcelDocument document = new ExcelDocument().open(new File("example/data/input/data-with-multiple-tabs.xlsx"));
+
+        for (String sheetName : document.getSheetNames()) {
+            DataReader reader = new ExcelReader(document).setFieldNamesInFirstRow(true).setSheetName(sheetName);
```
Comment (gvjoshi25) [inline src/main/java/com/northconcepts/datapipeline/examples/cookbook/ReadAllTabsFromAnExcelFile.java:23]: @chlocarlos Let's print `sheetName` as sysout here for better understanding.

**Review by gvjoshi25 — CHANGES_REQUESTED** [review]

File: src/main/java/com/northconcepts/datapipeline/examples/cookbook/ReadAllTabsFromAnExcelFile.java (hunk 2 above)
Comment (chlocarlos) [reply src/main/java/com/northconcepts/datapipeline/examples/cookbook/ReadAllTabsFromAnExcelFile.java:23]: @gvjoshi25 Done

**Review by gvjoshi25 — APPROVED** [review]
@DeleTaylor Please review.

**Review by DeleTaylor — APPROVED** [review]

**Review by DeleTaylor — CHANGES_REQUESTED** [review]
@chlocarlos please put a different dataset in the second tab (like countries or agents) and give both tabs names.  Right now it's very hard to see that the two tabs have different data and a developer might think we're printing the first tab twice.

**Review by DeleTaylor — APPROVED** [review]

**Pull request merged**